app/mocks/coda-responses/refresh-coda-data-for-tests.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * This is a script to refresh the mock return data for tests.
+ */
+
+import * as fs from 'fs'
+import * as https from 'https'
+import * as path from 'path'
+import * as toml from 'toml'
+import {URL} from 'url'
+
+export const questionIds = [8486]
+
+async function main(): Promise<void> {
+  const codaToken = readCodaToken()
+  await Promise.all(
+    questionIds.map(async (questionId) => {
+      const data = await getData(questionId, codaToken)
+      await writeFile(questionId, data)
+    })
+  )
+}
+
+const getData = async (questionId: number, codaToken: string) => {
+  const options = {
+    hostname: 'coda.io',
+    port: 443,
+    path: `/apis/v1/docs/fau7sl2hmG/tables/grid-sync-1059-File/rows?useColumnNames=true&sortBy=natural&valueFormat=rich&query=%22UI%20ID%22:%22${questionId}%22`,
```

Comment:
Thanks for the suggestion. I implemented the "extract a new method for making URL in both places" approach here: https://github.com/StampyAI/stampy-ui/pull/276/commits/e7cf3bb048fa8a933a5330d1d66f29c0977a1150

This was easier than exporting `makeCodaRequest` from `stampy.ts` because the TS compilation of `stampy.ts` expects injected CF global variables, which I thought would be tricky and confusing to do in the `refresh-coda-data-for-tests.ts` script.